scripts/update8.sh

```diff
@@ -24,7 +21,11 @@ set -euo pipefail
 echo "Common defs"
 
 # shellcheck disable=SC1091
-source import-common.sh
+modules=(corba langtools jaxp jaxws nashorn jdk)
+
+WORKSPACE="<Define your local workspace path>"
+GITHUB_IBM="Path for extension git repo ex : git@github.ibm.com:open-class-libraries"
+J9REPOSITORY="git repo name" 
```

Comment:
These variables should either be commented out or in some kind of help option. The current setup will override whatever is passed in.

Reply:
Hi, yes commented out and giving description would be useful.